android/src/newarch/CleverTapModule.kt

```diff
@@ -13,13 +13,10 @@ import com.facebook.react.bridge.ReadableMap
 @Suppress("unused")
 class CleverTapModule(reactContext: ReactApplicationContext?) : NativeCleverTapModuleSpec(reactContext) {
 
-    private val cleverTapModuleImpl: CleverTapModuleImpl
-    override fun getName(): String {
-        return CleverTapModuleImpl.REACT_MODULE_NAME
-    }
+    private val cleverTapModuleImpl: CleverTapModuleImpl = CleverTapModuleImpl(reactContext!!)
 
-    init {
-        cleverTapModuleImpl = CleverTapModuleImpl(reactContext!!)
+    override fun getName(): String {
+        return Constants.REACT_MODULE_NAME
     }
```

Comment:
what does this override do and why is it added?

Reply:
It is part of the generated abstract ```NativeCleverTapModuleSpec```. It was present in this class before, I've just moved it a few lines.

Reply:
@CTLalit RN by default generates the parent class `NativeCleverTapModuleSpec.java` and this method needs to be overriden